Netezos/Utils/Extensions/DateTimeExtension.cs

```diff
@@ -10,5 +10,10 @@ public static int ToUnixTime(this DateTime datetime)
         {
             return (int)(datetime - Epoch).TotalSeconds;
         }
+
+        public static DateTime FromUnixTime(this long seconds)
```

Comment:
The name of the method is a bit misleading...
````cs
var date = 1234.FromUnixTime();
````
sounds weird =)

I suggest either:
````cs
var date = DateTimeExtension.FromUnixTime(1234);
````
or 
````cs
var date = 1234.ToDateTime();
````